roles/horizon/templates/etc/openstack-dashboard/local_settings.py

```diff
@@ -109,9 +106,24 @@
 # including on the login form.
 #HORIZON_CONFIG["disable_password_reveal"] = False
 
+HORIZON_CONFIG = {
 {% if horizon.customize %}
-HORIZON_CONFIG["customization_module"] = "horizon-customization.horizon_customization"
+    'customization_module': 'horizon-customization.horizon_customization',
 {% endif %}
+    'dashboards': ('project', 'admin', 'settings',),
+    'default_dashboard': 'project',
+    'user_home': 'openstack_dashboard.views.get_user_home',
+    'ajax_queue_limit': 10,
+    'auto_fade_alerts': {
+        'delay': 3000,
+        'fade_duration': 1500,
+        'types': ['alert-success', 'alert-info']
```

Comment:
Okay, looks like those were upstream defaults, except for the `dashboards` line. Lets remove it all, and see what the default experience is like before we make adjustments.